flink-end-to-end-tests/test-scripts/test_sql_client.sh

```diff
@@ -85,6 +90,31 @@ function sql_cleanup() {
 }
 on_exit sql_cleanup
 
+function prepare_elasticsearch {
+  echo "Preparing Elasticsearch(version=$ELASTICSEARCH_VERSION)..."
+  # elastcisearch offers different release binary file for corresponding system since version 7.
+  case "$(uname -s)" in
+      Linux*)     OS_TYPE=linux;;
+      Darwin*)    OS_TYPE=mac;;
+      *)          OS_TYPE="UNKNOWN:${unameOut}"
+  esac
+
+  if [[ "$ELASTICSEARCH_VERSION" == 6 ]]; then
+    DOWNLOAD_URL=$ELASTICSEARCH6_DOWNLOAD_URL
+  elif [[ "$ELASTICSEARCH_VERSION" == 7 ]] && [[ "$OS_TYPE" == "mac" ]]; then
+    DOWNLOAD_URL=$ELASTICSEARCH7_MAC_DOWNLOAD_URL
+  elif [[ "$ELASTICSEARCH_VERSION" == 7 ]] && [[ "$OS_TYPE" == "linux" ]]; then
+    DOWNLOAD_URL=$ELASTICSEARCH7_LINUX_DOWNLOAD_URL
+  else
+    echo "Unsupported elasticsearch version($ELASTICSEARCH_VERSION) for OS: $OS_TYPE, fallback to elasticsearh($ELASTICSEARCH_VERSION)."
```

Comment:
What about echo an error message and `exist 1` to give an explicitly exception? Otherwise, we may miss the error alert if 7 doesn't work.

Reply:
okay，giving an exception is better than fallback to elasticsearch 6.